src/main/java/de/tum/in/www1/artemis/web/rest/metis/conversation/ConversationResource.java

```diff
@@ -167,4 +168,17 @@ private void checkEntityIdMatchesPathIds(Conversation conversation, Optional<Lon
             }
         });
     }
+
+    /**
+     * GET /conversations/for-notifications
+     *
+     * @return the list of Conversations for which the current user should receive notifications about
+     */
+    @GetMapping("/conversations/for-notifications")
```

Comment:
This URL does not follow REST standards

Reply:
I changed it to `api/courses/conversations-for-notifications`, is this fine?